x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/servicenow/servicenow.tsx

```diff
@@ -138,6 +138,7 @@ export function getServiceNowITSMActionType(): ActionTypeModel<
       return validationResult;
     },
     actionParamsFields: lazy(() => import('./servicenow_itsm_params')),
+    actionConnectorDropdownComponent: lazy(() => import('./servicenow_dropdown_row')),
```

Comment:
Current extension of the interface for `ActionTypeModel` has a big impact and need to have a good argumentation with providing a clear documentation for the developers of the new action types.
It feels like the name `actionConnectorDropdownComponent` doesn't say clearly about the usage of it.

From the PR implementation, it seems to be something like `customConnectorSelectItemComponent`. 
This is basically a sub component for the `ConnectorsDropdown`, which is the custom rendering for the Connector item.

Reply:
Great feedback, I updated it to `customConnectorSelectItemComponent` and added some docs to the README.